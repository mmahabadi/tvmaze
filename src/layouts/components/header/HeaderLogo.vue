<template>
  <a href="/">
    <img src="@/assets/logo.png" alt="logo" class="w-full h-12 object-contain" />
  </a>
</template>
