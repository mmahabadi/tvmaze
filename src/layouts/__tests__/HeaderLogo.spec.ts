import { mount, type VueWrapper } from '@vue/test-utils'
import { describe, expect, beforeEach, it } from 'vitest'
import HeaderLogo from '@/layouts/components/header/HeaderLogo.vue'

describe('HeaderLogo', () => {
  let wrapper: VueWrapper<unknown>

  beforeEach(() => {
    wrapper = mount(HeaderLogo, {
      global: {
        stubs: {
          RouterLink: true
        }
      }
    })
  })

  it('renders correctly', () => {
    expect(wrapper.html()).toMatchSnapshot()
  })
})
