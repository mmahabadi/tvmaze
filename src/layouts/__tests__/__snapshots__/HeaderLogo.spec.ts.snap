// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`HeaderLogo > renders correctly 1`] = `"<router-link-stub to="/" replace="false" custom="false" ariacurrentvalue="page"></router-link-stub>"`;
